src/di/binding.js

```diff
@@ -0,0 +1,29 @@
+import { camelCase } from 'lodash';
```

Comment:
can we rename `di` to something more sensible like `graphInjector`?

Reply:
`graph`

Reply:
totally graph!